prow/cmd/hook/githubagent.go

```diff
@@ -89,6 +91,9 @@ func (ga *GitHubAgent) handleIssueCommentEvent(ic github.IssueCommentEvent) {
 	if err := ga.commentTrigger(ic); err != nil {
 		l.WithError(err).Error("Error triggering after issue comment event.")
 	}
+	if err := ga.lgtmComment(ic); err != nil {
```

Comment:
Completely agree, I'd like to get this one in and then write a simple framework for future commands with this one and the test triggers as its guinea pig :)